src/main/java/seedu/address/logic/Messages.java

```diff
@@ -37,16 +37,15 @@ public static String getErrorMessageForDuplicatePrefixes(Prefix... duplicatePref
     public static String format(Person person) {
         final StringBuilder builder = new StringBuilder();
         builder.append(person.getName())
-                .append("\n IC: ")
+                .append("; NRIC: ")
                 .append(person.getIc())
-                .append("\n DOB: ")
+                .append("; DOB: ")
                 .append(person.getDob())
-                .append("\n Admitted: ")
-                .append(person.getAdmissionDate())
-                .append("\n Ward: ")
+                .append("; Ward No.: ")
                 .append(person.getWard())
-                .append("\n Tags: ")
-                .append(person.getTags());
+                .append("; Admission Date: ")
+                .append(person.getAdmissionDate())
+                .append("; Tags: ");
```

Comment:
Just curious, why was this changed? Is it due to the fact that we can fit everything in the screen?

Reply:
thanks for pointing this out, I just changed it back